### PR TITLE
DI-547 Specified opening time bug fix test

### PR DESCRIPTION
**Pull request by mik3ola**
# Warning

Please don't modify this description yet it will be populated once you create the pull request.

### CODE REVIEW

File: test/integration/steps/utilities/change_event_builder.py [hunk 1]
```diff
@@ -283,4 +275,16 @@ def set_opening_times_change_event(service_type: str) -> ChangeEvent:
             "IsOpen": False,
         }
     )
+    change_event.specified_opening_times.append(
```
Comment (JackPlowman) [inline test/integration/steps/utilities/change_event_builder.py:278]: This would break other tests as the opening times should be marked as illogical if this is received in a change event

File: test/integration/steps/utilities/change_event_builder.py [hunk 2]
```diff
@@ -104,14 +104,6 @@ def build_change_event_from_default(self):
                     "AdditionalOpeningDate": "Dec 24 2022",
                     "IsOpen": True,
                 },
-                {
```
Comment (JackPlowman) [inline test/integration/steps/utilities/change_event_builder.py:107]: Why is this removed?

File: test/integration/features/F006_Opening_times.feature [hunk 3]
```diff
@@ -90,3 +90,10 @@ Feature: F006. Opening times
       | opening_type |
       | General      |
       | Additional   |
+
+  @complete @pharmacy_no_log_searches @wip
```
Comment (JackPlowman) [inline test/integration/features/F006_Opening_times.feature:94]: Please remove `@wip`

**Review by JackPlowman — CHANGES_REQUESTED** [review]
Functionality looks good, just interested in why a couple of bit have been changed as they might break other tests

File: test/integration/steps/utilities/change_event_builder.py (hunk 1 above)
Comment (mik3ola) [reply test/integration/steps/utilities/change_event_builder.py]: Change now moved to step def file.

File: test/integration/features/F006_Opening_times.feature (hunk 3 above)
Comment (mik3ola) [reply test/integration/features/F006_Opening_times.feature]: Resolved
